test/cases/image_tests.sh

```diff
@@ -85,8 +87,10 @@ cd $WORKING_DIRECTORY
 OSBUILD_LABEL=$(matchpathcon -n "$(which osbuild)")
 sudo chcon "$OSBUILD_LABEL" /usr/libexec/osbuild-composer-test/image-info
 
+get_test_cases
 # Run each test case.
-for TEST_CASE in $(get_test_cases); do
+for TEST_CASE in "${TEST_CASES[@]}"; do
```

Comment:
I don't understand why not keep using the function like before? Surely the function itself can return an appropriate result, no ?